### PR TITLE
controller successfully created

### PR DESCRIPTION
**Pull request by anujsingh503**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -25,10 +25,10 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-data-jpa</artifactId>
 		</dependency>
-		<dependency>
+		<!-- <dependency>
```
Comment (anoop-rana) [inline pom.xml:28]: thik kr

**Review by anoop-rana — COMMENTED** [review]
please close the open thread

File: pom.xml (hunk 1 above)
Comment (anujsingh503) [reply pom.xml:28]: kr dunga

**Comment by anujsingh503** [comment]
fixed

**Comment by anoop-rana** [comment]
reopened pr

**Review by anoop-rana — APPROVED** [review]
Approved

**Review by anoop-rana — APPROVED** [review]
Approved

**Pull request merged**